.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/landusedata"]
```

Comment:
I suggest adding a header to explain what is going on here. Adapting the CTSM header I suggest something like this...

``` yaml
# This is a git submodule file to include additional submodules when FATES is checkout out.
#
# It includes optional additional support for
# git-fleximod (https://github.com/ESMCI/git-fleximod)
# that will be used when git-fleximod is used to check it out (i.e. for CESM)
#
# The additional flags supported by git-fleximod are
# fxtag - the tag associated with the submodule, this tag can be tested for
#         consistancy with the submodule hash using git-fleximod status
#         the hash can be updated to the tag using git-fleximod update
#
# fxrequired - indicates if a given submodule should be checked out on install
#              submodules can be toplevel or internal and required or optional
#              toplevel means that the submodule should only be checked out if the
#              module is the toplevel of the git repo (is not a submodule itself)
#              internal means that the submodule is needed by the component whether
#              the component is toplevel or the submodule of another repo
#              required means that the submodule should always be checked out
#              optional means that the submodule should only be checked out if the
#              optional flag is provided to git-fleximod or the submodule name is
#              explicitly listed on the git-fleximod command line.
#
# fxsparse -   this is a path to a git sparse checkout file indicating that the
#              submodule should be checked out in sparse mode
#
# fxDONOTUSEurl    -   this field is used by git-fleximod test to insure that the url is pointing
#              to the official url of the repo and not to an unofficial fork.
#              It is intended for use of github workflows to test commits to protected
#              repository branches.
#
```

Reply:
Thanks @ekluzek, I've added a trucated version of this.